tests/06-sim-actions.tests

```diff
@@ -12,7 +12,7 @@ test type: bpf-sim
 06-sim-actions	all		write		1		0x856B008	N	N	N	N	ERRNO(1)
 06-sim-actions	all		close		4		N		N	N	N	N	TRAP
 06-sim-actions	all		openat		0		0x856B008	4	N	N	N	TRACE(1234)
-06-sim-actions	all		fstat		N		N		N	N	N	N	KILL_PROCESS
+06-sim-actions	all		fstatfs		4		0x856B008	N	N	N	N	KILL_PROCESS
 06-sim-actions	all		rt_sigreturn	N		N		N	N	N	N	LOG
 06-sim-actions	x86		0-2		N		N		N	N	N	N	KILL
 06-sim-actions	x86		7-107		N		N		N	N	N	N	KILL
```

Comment:
Previously the test used `fstat` which has a syscall number of 108 on x86, hence the tests only go up to 107 on x86; with the test now using `fstatfs`, which is 100 on x86` this test line needs to be updated.

Reply:
There is a similar issue below with x86_64 with `fstat` being 108 and `fstatfs` being 100.